Source/WebKit/UIProcess/gtk/AcceleratedBackingStoreDMABuf.cpp

```diff
@@ -377,10 +377,15 @@ void AcceleratedBackingStoreDMABuf::BufferSHM::didUpdateContents()
 {
 #if USE(CAIRO)
     m_surface = m_bitmap->createCairoSurface();
-    cairo_surface_set_device_scale(m_surface.get(), m_deviceScaleFactor, m_deviceScaleFactor);
 #elif USE(SKIA)
-    notImplemented();
+    m_surface = cairo_image_surface_create_for_data(static_cast<unsigned char*>(m_bitmap->data()), CAIRO_FORMAT_ARGB32, m_size.width(), m_size.height(), m_bitmap->bytesPerRow());
```

Comment:
More of a question than a comment, but is `CAIRO_FORMAT_ARGB32` the only format that both Skia and Cairo ever deal with in the UI process?

Reply:
Yes.

Reply:
Thanks. `R-b=me` then